scripts/deploy_verdaccio.sh

```diff
@@ -1,28 +1,33 @@
-CATALOGS=(_utils pwa serverless universal firebug)
-
-echo "//localhost:4873/:_authToken=\"CjmKyL6UDkX6FDpNnP64fw==\"" >> ~/.npmrc
+CATALOGS=(_utils serverless pwa universal firebug)
+echo "//localhost:4873/:_authToken=\"CjmKyL6UDkX6FDpNnP64fw==\"" >>~/.npmrc
 
 npm install -g @angular/cli
 npm install -g verdaccio
-verdaccio --config scripts/default.yaml >> verdacio_output &
+verdaccio --config scripts/default.yaml >>verdacio_output &
 sleep 5
 head verdacio_output
-npm set registry=http://localhost:4873/
+npm set registry http://localhost:4873/
 
 exitStatus=0
 
+center() {
+    termwidth="$(tput cols)"
+    padding="$(printf '%0.1s' ={1..500})"
+    printf '\e[1;44m%*.*s %s %*.*s\e[0m\n' 0 "$(((termwidth - 2 - ${#1}) / 2))" "$padding" "$1" 0 "$(((termwidth - 1 - ${#1}) / 2))" "$padding"
+}
+
 cd schematics
 
-for i in "${CATALOGS[@]}"
-do :
-    echo publishing $i
+for i in "${CATALOGS[@]}"; do
+    center "$i"
     cd $i
-    npm install
-    echo 'INSTALLED INSTALLED'
-    ls node_modules/@ng-toolkit/_utils/testing
-    if ./publish_verdaccio.sh
-    then
-        echo $i passed
+    npm install --verbose
```

Comment:
I think that this is not necessary, as long as particular packages scripts take care about that. I think I added this line for debugging purposes.

Reply:
You mean the `npm install` part? I do find helpful to echo the current package being installed and the verbose mode was for debugging purpose, to check out if there was anything bad installed with the package dependencies. At the end, everything was okay.

Also ignore the center thing, it is to divide sections, so we can scroll or jump to any package step by looking at the colorful header.